pegasus/sites.v3/code.org/public/educate/csd.md.erb

```diff
@@ -238,7 +238,7 @@ For answers to our most frequently asked questions, click on the questions below
 	<summary style="font-size: 18px"><b>Is CS Discoveries mapped to standards?</b></summary>
 	<p>
 	<br/>
-CS Discoveries was written using both the K-12 Framework for Computer Science and the 2017 CSTA standards as guidance. Currently, every lesson in CS Discoveries contains mappings to the relevant 2017 CSTA standards. The summary of of all CSTA 2017 mappings can be found at <a href="https://studio.code.org/courses/csd/standards">studio.code.org/courses/csd-2021/standards</a>. A google sheet version of the standards can be found at <a href="https://docs.google.com/spreadsheets/d/18Wk1pI8bZly4P0Rq2OuSXMzlwWzuxhnroYmxiYNaj6c/edit?usp=sharing", target=_"blank">CSD Standards 2021</a>.
+CS Discoveries was written using both the K-12 Framework for Computer Science and the 2017 CSTA standards as guidance. Currently, every lesson in CS Discoveries contains mappings to the relevant 2017 CSTA standards. The summary of of all CSTA 2017 mappings can be found at <a href="https://studio.code.org/courses/csd/standards">studio.code.org/courses/csd-2022/standards</a>. A google sheet version of the standards can be found at <a href="https://docs.google.com/spreadsheets/d/18Wk1pI8bZly4P0Rq2OuSXMzlwWzuxhnroYmxiYNaj6c/edit?usp=sharing", target=_"blank">CSD Standards 2021</a>.

```

Comment:
The text says `blank>CSD Standards 2021</a>` Should we remove the year? or does this point to the spreadsheet for 2021 and does curriculum need to update?

Reply:
I don't know, but my best guess is that curriculum is not done generating this info for 2022 yet, since they only just got the standards spreadsheets for 2022 last week. so I think there is probably nothing to do here now, and they can updated it once the 2022 sheet is ready.